apps/api/src/user/user.controller.ts

```diff
@@ -27,9 +28,20 @@ export class UserController {
 
   @Post('/invite')
   async invite(@Body() data: CreateUserInviteDTO, @Req() req: IRequest): Promise<UserRO> {
-    const tokenUser = req.user;
-    const user = await this.userService.createUserFromInvite(data, tokenUser);
-    return new UserRO(user);
+    try {
+      const tokenUser = req.user;
+      const user = await this.userService.createUserFromInvite(data, tokenUser);
+      return new UserRO(user);
+    } catch (err) {
+      if (
+        err instanceof QueryFailedError &&
+        err.message.includes('duplicate key value violates unique constraint')
+      ) {
+        throw new UnprocessableEntityException('User already exists');
```

Comment:
I'd suggest using ConflictException when the user already exists. StatusCode - 409